pdb2sql/StructureSimilarity.py

```diff
@@ -658,6 +662,8 @@ def compute_irmsd_pdb2sql(self,cutoff=10,method='svd',izone=None,exportpath=None
         Returns:
             float: i-RMSD value of the conformation
         """
+        backbone = ['CA', 'C', 'N', 'O']
```

Comment:
should we make that into a class variable ? so that we do not have to redefine it each time

Reply:
Good idea! will make one.